### ISSUE TITLE
Lighting on DM-Gothic map

### ISSUE THREAD

**Issue by chris-mcc** [dpjudas/UT99VulkanDrv#5]
I've been using the renderer for a few days and have only really noticed an issue around the lighting from the lanterns in the DM-gothic.

I'm running UT99 with the 469B patch with the textures from https://sites.google.com/view/unrealhdtextures/home including HD skins.

They're emitting out too much and can be a little annoying at times.  To me the dx11 renderer does this bit correct.  Sometimes there can be twop light sources in front but a third one in the back can shine out more,

Also on this map it looks like the purple sky light source isn't being picked up just quite enough at times - minor minor point, but maybe relevent.

**Comment by chris-mcc** [comment]
Hey, i'm using the release version.  Oh that sounds neat.  If you can cut a release I can trial out the changes on the map :)

Now that you mention it - I vaguely remember the original dx renderer way back then behaving similarly (there is only a slight difference between the new different renderers (xopengl, directx11) as far i can tell it's approx like <=15%).

**Comment by chris-mcc** [comment]
Thanks,

There does seem to be better mood lighting to my eyes - ie it gets darker in certain parts - if that's expected.

There's still too much lens glare I think - but I think it has improved in game.
![vulkan14renderer](https://user-images.githubusercontent.com/11029841/178364864-eba13723-f16e-46e3-856d-9f38fcd6e643.jpg)
eplay.
![dx11Renderer](https://user-images.githubusercontent.com/11029841/178364852-2080f6fd-a309-439c-aa47-b3d257319630.jpg)

I've attached what the dx11 renderer shows versus the vulkan to illustrate the difference.

**Comment by dpjudas** [comment]
Uploaded a 1.5 version where the coronas should be a bit better. The colors are also slightly stronger, but not to the same degree as the dx11 renderer. I'm not sure why it differs - maybe that renderer enhances the textures in some way? - I'm not sure.

This 1.5 version is as close as I can get it to look like that without starting to seriously diverge from my goal of getting this renderer to render exactly the same as the original D3D renderer did.

**Comment by chris-mcc** [comment]
Mm, the dx11 / xopengl driver do renderer generally darker but hit the same level of darkness.

I think 1.5 is very close at this point.

Does the renderer support the S3TC=TRUE flag?

I know the dx11 renderer adds extra postprocessing like bloom and ParallaxOcclusionMapping btw.

Good job

**Comment by dpjudas** [comment]
I'm not 100% sure. The texture upload code does have support for S3TC and a number of other texture formats added by 469.

I'm also not sure how those newer renderers support parallax occlusion mapping as all light in UT is baked into the lightmaps and that means it doesn't have the info required to do such an effect. Maybe they are accessing the dynamic light list directly?

**Comment by chris-mcc** [comment]
Not sure; sounds like a bit of post processing in the dx 11 renderer in general - I think the xopengl doesn't do parallax occlusion mapping or as much post processing.

I tried enabling the UseS3TC=1 but didn't really see a difference but noticed a glitch in the face 2 map - above the main entrance.  It's meant to show a bit of the backdrop but shows white instead.  So I disabled it.

Did come across an old glitch that I've seen in all renderers on the last stand map.
![glitchVulkan15Renderer](https://user-images.githubusercontent.com/11029841/178552698-9d4bbff1-7dcf-47f5-85f4-6b082a1a9613.jpg)

those top corners glitch like off/on for me.
